virt/virt-2-6-release-notes.adoc

```diff
@@ -65,6 +65,15 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 === Web console
 
 //CNV-9312: The VM wizard has been completely redesigned, simplifying VM creation
+The *Create Virtual Machine Template* wizard enables you to:
```

Comment:
This should have a bullet point. The two subsequent bullet points should then be second level ( ** )